### PR TITLE
A proposal to indicate how to delete the RStudio "instance"

### PR DESCRIPTION
**Pull request by yvanlebras**

### CODE REVIEW

File: topics/galaxy-ui/tutorials/rstudio/tutorial.md [hunk 1]
```diff
@@ -346,6 +346,10 @@ Here is one last bonus we will mention about RStudio. It's difficult to remember
 >
 {: .hands_on}
 
+## Stopping RStudio
+
+If you want to stop your RStudio interactive tool, once you finish your R treatment and if relevant, saving your RStudio environment in your Galay history, you just have to delete the corresponding history dataset named `RStudio` and showing a in progress state, so yellow.
```
Comment (bgruening) [inline topics/galaxy-ui/tutorials/rstudio/tutorial.md]: You could also mention the overview page at: `User → Active InteractiveTools.` 

And maybe explicitly that data is lost that is not trasferred to Galaxy into a persistent storage.

Comment (yvanlebras) [reply topics/galaxy-ui/tutorials/rstudio/tutorial.md]: oh yeah! eagle eye !

Comment (bebatut) [reply topics/galaxy-ui/tutorials/rstudio/tutorial.md]: Could you put the explanation as a snippet (then we can call it from the 4 R tutorials)? Thanks a lot

Comment (yvanlebras) [reply topics/galaxy-ui/tutorials/rstudio/tutorial.md]: True!

Comment (bgruening) [reply topics/galaxy-ui/tutorials/rstudio/tutorial.md]: ping

**Pull request merged**